docs/iris/src/whatsnew/latest.rst.template

```diff
@@ -10,6 +10,12 @@ This document explains the changes made to Iris for this release
    :depth: 3
 
 
+Announcements
+=============
+
+* N/A
+
+
```

Comment:
Added this new section to the `template`.

Ordering of sections is important, and IMHO `Announcements` should go first :wink: